crypto/asn1/a_gentm.c

```diff
@@ -157,10 +157,10 @@ int ASN1_GENERALIZEDTIME_print(BIO *bp, const ASN1_GENERALIZEDTIME *tm)
     if (BIO_printf(bp, "%s %2d %02d:%02d:%02d%.*s %d%s",
                    _asn1_mon[M - 1], d, h, m, s, f_len, f, y,
```

Comment:
Another way is to declare this thing statically into a header file, such as `asn1_locl.h`. But no idea if this follows the design principles of the project...